static/locales/en-US.yaml

```diff
@@ -405,6 +405,10 @@ Settings:
     Download Behavior: Download Behavior
     Download in app: Download in app
     Open in web browser: Open in web browser
+  Experimental Settings:
+    Experimental Settings: Experimental Settings
+    Warning: These settings are experimental, enabling them may result in a worse FreeTube experience. Use at your own risk!
```

Comment:
If you say it has a potential for crashes and recommend the data is backed up then I would be fine with approving as is. It is an experimental setting after all

Reply:
Updated the message now.